src/index.js

```diff
@@ -18,14 +19,16 @@ async function start() {
   const instance_id = wait_instance_response.start.instance_id
   core.info(`${provider} instance started with ID: ${instance_id}`)
 
-  setOutput(start_instance_response.runner_name)
+  const runner_id = await waitForRunnerRegistered(
+    start_instance_response.runner_name
+  )
 
-  await waitForRunnerRegistered(start_instance_response.runner_name)
+  setOutput(start_instance_response.runner_name, runner_id)
 }
 
 async function stop() {
   const stop_instance_response = await slab.terminateInstanceRequest(
-    config.input.label
+    config.input.runnerId
   )
   await slab.waitForInstance(stop_instance_response.task_id, 'stop')
```

Comment:
that's fully breaking, is that what we discussed ?

Reply:
It's fully breaking but user of older version of the action can still be used. It's breaking from action point of view but Slab handle the backward compatibility.